package.json

```diff
@@ -38,6 +38,5 @@
   "homepage": "https://github.com/urbanairship/urbanairship-accengage-cordova.git",
   "bugs": {
     "url": "https://github.com/urbanairship/urbanairship-accengage-cordova/issues"
-  },
-  "homepage": "https://github.com/urbanairship/urbanairship-accengage-cordova#readme"
+  }
```

Comment:
I suggest you add "accengage" to the "keywords" list in package.json

Reply:
ok